engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -232,15 +232,41 @@ private void initRenderingSupport() {
     private void initRenderGraph() {
         RenderGraph renderGraph = new RenderGraph();
 
-        // ShadowMap generation
+        addShadowMapNodes(renderGraph);
+
+        addReflectionNodes(renderGraph);
+
+        addSkyNodes(renderGraph);
+
+        addWorldRenderingNodes(renderGraph);
+
+        addLightingNodes(renderGraph);
+
+        addRefractiveReflectiveNode(renderGraph);
+
+        add3dDecorationNodes(renderGraph);
+
+        addPrePostProcessingNodes(renderGraph);
+
+        addPostProcessingNodes(renderGraph);
+
+        addImageToSceeenNodes(renderGraph);
```

Comment:
Yes, I'm still aware of that issue - however, I'm waiting for shader overhaul before I start splitting shaders, and consequently, Nodes.